terraform/resources/aca/provider.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  required_providers {
+    docker = {
+      source  = "kreuzwerker/docker"
```

Comment:
So, I think we're both going to want to migrate this docker setup to use a provider that doesn't require the docker daemon. I've had some discussions with STLTs that don't want to or don't have docker on their workers.